model_compression_toolkit/gptq/keras/quantizer/gumbel_rounding/base_gumbel_rounding.py

```diff
@@ -31,21 +31,29 @@
 P_INIT = 0.01
 
 
-def _init_aux_var(w_shape: List[int], m: int, p: float = P_INIT) -> np.ndarray:
+def init_aux_var(ceil_indicator: np.ndarray, w_shape: List[int], m: int, p: float = P_INIT) -> np.ndarray:
     """
     This function generate a random pi matrix for Gumbel Rounding
```

Comment:
I would add a sentence about the initialization process (the purpose of checking the ceil of the original value, the shift purpose...)